core/src/main/java/de/danoeh/antennapod/core/util/playback/Timeline.java

```diff
@@ -68,7 +68,7 @@ public Timeline(Context context, ShownotesProvider shownotesProvider) {
 
     private static final Pattern TIMECODE_LINK_REGEX = Pattern.compile("antennapod://timecode/((\\d+))");
     private static final String TIMECODE_LINK = "<a class=\"timecode\" href=\"antennapod://timecode/%d\">%s</a>";
-    private static final Pattern TIMECODE_REGEX = Pattern.compile("\\b(?:(?:(([0-9][0-9])):))?(([0-9][0-9])):(([0-9][0-9]))\\b");
+    private static final Pattern TIMECODE_REGEX = Pattern.compile("\\b(?:(?:([01]?\\d|2[0-3]):)?([0-5]?\\d):)?([0-5]?\\d)\\b");
```

Comment:
This might be a silly question, but what happens to files with durations > 24 hours?

Reply:
I'm not sure, didn't think of that to be honest. I'll test and switch up the regex if needed.

Reply:
OK should be fixed now.